frontend/src/metabase/query_builder/components/view/ViewHeader/components/DashboardBackButton/DashboardBackButton.module.css

```diff
@@ -0,0 +1,3 @@
+.DashboardBackButton {
+  border-color: color-mix(in srgb, var(--mb-color-border), black 5%) !important;
```

Comment:
(Might need to check in dark themed apps if this would work)

Reply:
I'll do that in a separate PR so it's not mangled in with the rest of this